### PR TITLE
SSH Keys: put SSH needed configuration for separate SSH keys

### LINKED ISSUE #304: SSH: document usage of .ssh/config file for custom ssh key
We already have a bit of (outdated) documentation here: https://www.clever-cloud.com/doc/get-help/troubleshooting/#-i-class-icon-double-angle-right-i-ssh-password-is-asked

IMHO we should move that part to the SSH configuration page and add a link to the section into the troubleshooting page

### PR DESCRIPTION
**Pull request by BlackYoup**
Fixes #304

### CODE REVIEW

File: commons/admin-console/ssh-keys.md [hunk 1]
```diff
@@ -121,4 +121,34 @@ To check if your ssh key is correctly configured, you can try to run:
...
+> git@push-par-clevercloud-customers.services.clever-cloud.com: Permission denied (publickey).
+>
+> fatal: Could not read from remote repository.
+>
+> Please make sure you have the correct access rights
+> and the repository exists.
+
+An error can occur when your SSH agent has not be configured to use your SSH key.
+
+Most of the time, it's due to the presence of multiple ssh key, generated by 3rd party softwares, like Github for Mac.
```
Comment (cnivolle) [inline commons/admin-console/ssh-keys.md]: "multiple ssh key" => "multiple ssh keys"

File: commons/admin-console/ssh-keys.md [hunk 2]
```diff
@@ -121,4 +121,34 @@ To check if your ssh key is correctly configured, you can try to run:
...
+
+```bash
+# config for Clever Cloud specific key (Paris zone)
+Host ccloud-par
+  Hostname push-par-clevercloud-customers.services.clever-cloud.com
+  User git
+  IdentityFile ~/.ssh/id_rsa_clevercloud
+
+# config for Clever Cloud specific key (Montreal zone)
+Host ccloud-par
```
Comment (cnivolle) [inline commons/admin-console/ssh-keys.md]: "Host ccloud-par" => "Host ccloud-mtl"

**Comment by BlackYoup** [comment]
Updated :)

File: commons/admin-console/ssh-keys.md [hunk 3]
```diff
@@ -121,4 +121,34 @@ To check if your ssh key is correctly configured, you can try to run:
...
+> git@push-par-clevercloud-customers.services.clever-cloud.com: Permission denied (publickey).
+>
+> fatal: Could not read from remote repository.
+>
+> Please make sure you have the correct access rights
+> and the repository exists.
+
+An error can occur when your SSH agent has not be configured to use your SSH key.
+
+Most of the time, it's due to the presence of multiple SSH keys generated by 3rd party softwares, like Github for Mac.
```
Comment (divarvel) [inline commons/admin-console/ssh-keys.md]: s/softwares/software/

**Comment by cnivolle** [comment]
@BlackYoup @divarvel I've fixed the typo. 
Everything's okay for you ?

**Pull request merged**